Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -75,6 +81,12 @@
             "value" : "🦄 Mikaela Caron - Maintaineur"
           }
         },
+        "hi" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "🦄 मिकाएला कैरोन - अनुरक्षक"
```

Comment:
Is this common to also change my name? and why does Google Translate translate this as "escort?" `अनुरक्षक`

Reply:
The Hindi script, also called Devanagari is capable of translating names as well. I'm not sure why Google is translating maintainer to escort in English; I think they mean 'protector'. The synonym is actually "maintainer", "care taker". If you'd like, I can keep it in English.

Reply:
@anupdsouza I'll put it up to your best judgement, if you think it the word makes sense in this context, then it's fine to leave it

Reply:
I've updated it to a more suitable translation